bendigobank.rb

```diff
@@ -0,0 +1,90 @@
+require 'watir'
+require 'json'
+require 'nokogiri'
+require 'date'
+require_relative 'account.rb'
+require_relative 'transaction.rb'
+
+class Bendigobank
+  URL = "http://demo.bendigobank.com.au"
+
+  def self.execute
+    connect
+    fetch_accounts
+    fetch_transactions
+  end
+
+  def self.connect
+    Watir.logger.ignore :deprecations
+    @browser = Watir::Browser.new :firefox
+    @browser.goto URL
+    @browser.button(name: 'customer_type').click
+  end
+
+  def self.fetch_accounts
+    html = Nokogiri::HTML.fragment(@browser.ol(class: "grouped-list grouped-list--compact").html)
+    parse_accounts(html)
+  end
+
+  def self.fetch_transactions
+    to_date = Time.new().to_datetime
+    from_date = to_date << 2
+    if from_date.month < 10
+      from_date = from_date.day.to_s + "/" + "0" + from_date.month.to_s + "/" + from_date.year.to_s
+    end
+    if to_date.month < 10 && to_date.day < 10
+      to_date = "0" + to_date.day.to_s + "/" + "0" + to_date.month.to_s + "/" + to_date.year.to_s
+      from_date = "0" + from_date.day.to_s + "/" + from_date.month.to_s + "/" + from_date.year.to_s
+    elsif to_date.month < 10
+      to_date = to_date.day.to_s + "/" + "0" + to_date.month.to_s + "/" + to_date.year.to_s
+    elsif date.day < 10
+      to_date = "0" + to_date.day.to_s + "/" + to_date.month.to_s + "/" + to_date.year.to_s
+      from_date = "0" + from_date.day.to_s + "/" + from_date.month.to_s + "/" + from_date.year.to_s
+    end
+     @browser.ol(class: 'grouped-list__group__items').each do |li|
+       li.a(class: 'panel--hover').click
+       @browser.i(class: 'ico-nav-bar-filter_16px').click
+       @browser.a(class: 'panel--bordered__item').click
+       @browser.ul(class: 'radio-group').li(index: 8).click
+       @browser.text_field(name: 'toDate').set(to_date)
+       @browser.text_field(name: 'fromDate').set(from_date)
+       @browser.button(class: 'button--primary').click
+       @browser.button(class: 'button--primary').click
+       while @browser.div(class: '_3Wd5wOSiEN').present? != true
```

Comment:
Method `.present?` returns boolean value no need to use operators `!=` or `==`
You can use the keyword `until condition` instead of `while !condition`